src/components/Choropleth.vue

```diff
@@ -5,6 +5,7 @@
             ref="map"
             style="height: calc(100vh - 48px); width: 100%"
             @update:bounds="waitingForMapBounds = false"
+            @ready="() => updateMap(selectedFeatures)"
```

Comment:
when you navigate to unknown country and then click home page, the map was not loading, we just needed an update when the map is ready basically